lib/netstd/Thrift/Transport/Client/THttpTransport.cs

```diff
@@ -134,6 +129,11 @@ public override void Close()
 
         public override async Task WriteAsync(byte[] buffer, int offset, int length, CancellationToken cancellationToken)
         {
+            if (_isDisposed)
```

Comment:
I'd recommend to put that into a method. The CheckDisposed() pattern is already used in some other files.

Reply:
Refactoring code is still a good thing. But not in the same ticket :-)

Reply:
I'm all for that. I'll look through all the IDisposable classes and and create a ticket to standardize them separately. If there are any other cleanup items that the maintainers want done but don't have time for then feel free to open some thrift tickets and I'll check them out and see how I can help.